public/app/core/components/DeleteButton/DeleteButton.tsx

```diff
@@ -0,0 +1,78 @@
+import React, { Component } from 'react';
+
+export default class DeleteButton extends Component<any, any> {
```

Comment:
- This can be made a `PureComponent` because its rendering depends entirely on state and props.
- It's easy to add `DeleteButtonProps` and `DeleteButtonState` types here, so we should do it, see [this guide](https://github.com/piotrwitek/react-redux-typescript-guide#stateful-components---class)